modules/view-recently-detected-vulnerabilities.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_module-type: CONCEPT
+[id="view-recently-detected-vulnerabilities_{context}"]
+= Viewing recently detected vulnerabilities
+
+The *Recently Detected Vulnerabilities* widget on the *Vulnerability Management* view shows a list of recently discovered vulnerabilities in your scanned images, based on the scan time and CVSS score.
+It also includes information about the number of images affected by the CVE and its impact (percentage) on your environment.
+
+* When you hover over a CVE in the list, you see an overview of the CVE, which includes scan time, CVSS score, description, impact, and whether it's scored by using CVSS v2 or v3.
```

Comment:
@gaurav-nelson Just one it's -> it is that got missed. I'm going to go ahead and merge this PR though if you can tag that onto another one of your PRs if you don't mind.